Frameworks/Foundation/NSOperation.mm

```diff
@@ -77,41 +64,85 @@ + (BOOL)automaticallyNotifiesObserversForKey:(NSString*)key {
     return NO;
 }
 
-/**
- @Status Interoperable
-*/
-+ (id)allocWithZone:(NSZone*)zone {
-    NSOperation* ret = [super allocWithZone:zone];
+- (void)_checkReady {
```

Comment:
Sycned up offline with John. Since this is an urgent fix, will follow-up on this separately.  #624

---
In reply to: [69982783](https://github.com/Microsoft/WinObjC/pull/602#discussion_r69982783) [](ancestors = 69982783)